pkgs/by-name/mo/moonlight/package.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  stdenv,
+  nodejs,
+  pnpm_9,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "moonlight";
+  version = "1.3.8";
+
+  src = fetchFromGitHub {
+    owner = "moonlight-mod";
+    repo = "moonlight";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-x/t6EcKB/fNO59v2fchxbXQVRcoHuXdlUNCtDlciep8=";
+  };
+
+  nativeBuildInputs = [
+    nodejs
+    pnpm_9.configHook
+  ];
+
+  pnpmDeps = pnpm_9.fetchDeps {
+    inherit (finalAttrs) pname version src;
+
+    hash = "sha256-amzSuaOtrusm6QREoDIfRvg5BV9hqrYKIMosYlx+/IQ=";
+  };
+
+  buildPhase = ''
+    runHook preBuild
+
+    pnpm run build
+
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    cp -r dist $out
+
+    runHook postInstall
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    description = "Yet another Discord mod, focused on providing a decent user and developer experience";
```

Comment:
please make description confirm with guidelines

Reply:
Yet another Discord mod isn't super appropriate. Maybe something like

"Discord mod to enhance user and developer experience" would be better